src/animad-keeper/animad-keepers-list.html

```diff
@@ -263,8 +274,16 @@ <h2>[[t('modal_delete_header')]]</h2>
         }
 
         // proteceted methods
-        _handleNew() {
-          console.log("_handleNew");
+        _handleNew(event) {
+          console.log("_handleNew", event);
+          window.location = '/keepers-view/create'
+          // TBD
+        }
+
+        _editRow(event){
```

Comment:
Wenn ich das richtig verstehe, dann machst du hier den Absprung auf eine Edit Seite. Warum nutzt du hier nicht die Polymer Komponenten?

Reply:
Meinst du sowas wie 'this.set('route.path', '/')? Das sollte ich ändern, stimmt.